docs/gettingstarted/query_test_report.rst

```diff
@@ -20,9 +20,14 @@ with default format fields. To see a list of all format fields, click :ref:`here
 .. note::
    ``buildtest rt`` is an alias for ``buildtest report`` command.
 
-Color Options (``buildtest report --color``)
+Color output of buildtest report (buildtest report --color)
 ---------------------------------------------
```

Comment:
you need the `----` to go all the way to length of the heading i think if you dont do that their is some rendering issues. Take a look at https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings

Reply:
-- added